frontend/src/modules/Traces/LatencyModalForm.tsx

```diff
@@ -13,13 +13,22 @@ const LatencyModalForm: React.FC<LatencyModalFormProps> = ({
 	onCancel,
 	latencyFilterValues,
 }) => {
-	const [form] = Form.useForm();
+    const [form] = Form.useForm();
+    const [disabled, setDisabled] = useState<boolean>(false)
+    const [min, setMin] = useState<number>(parseInt(latencyFilterValues.min))
+    const [max, setMax] = useState<number>(parseInt(latencyFilterValues.max))
+    
+    useEffect(() => {
+        min! > max! ? setDisabled(true) : setDisabled(false)
```

Comment:
Just curious, what's the use of `!` here ? What will be the output of `min!`? 

Also you can update it to be more like: `setDisabled(condition)`

Reply:
> 
> 
> <img alt="Screenshot 2021-06-05 at 7 35 11 PM" width="642" src="https://user-images.githubusercontent.com/12460410/120894283-39c96b80-c635-11eb-9370-23b877191e0b.png">
> 
>     1. Text overflowing in screen of width 1440px
> 
>     2. Also, can you change the color of the text to `rgb(227,38,54)`?

I'm not sure why but this fine in here for me, and also there's a typo there I'll fix that 😓  
![image](https://user-images.githubusercontent.com/43727167/120900346-f4676700-c651-11eb-9ff3-d25c54635ca9.png)

Reply:
So yeah, I was like writing like handleChange(onChange in `InputNumber`) function which also takes a null value as argument, basically, it says that the value will not be a null, and I forgot to remove that operator, will make changes